site/content/docs/private-locations/_index.md

```diff
@@ -11,12 +11,18 @@ aliases:
 cli: true
 ---
 
-A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. A Private Location can be used to run checks on your services which are not directly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
+A Private Location is a monitoring location that you set up and manage within your own network. The key difference is that you install a lightweight Checkly Agent on your own server or infrastructure.
```

Comment:
@checklyalex necessarily "within your internal network"? Later on we mention these can be spun up from different parts of the world, for example...

Reply:
Ok I will change to "A Private Location is a monitoring location that you manage by simply deploying a lightweight Checkly Agent."